src/platform/device.gni

```diff
@@ -25,7 +25,7 @@ declare_args() {
   chip_fake_platform = false
 
   # Include wifi-paf to commission the device or not
-  chip_device_config_enable_wifipaf = false
+  chip_device_config_enable_wifipaf = current_os == "linux"
```

Comment:
why did we change this default? Could you add a comment here on sideffects?

I am not familiar with PAF - do we expect any linux device with a wifi card to support this or do you require special hardware/drivers? Is this expected to be transparent for devices that do not support it?

Reply:
This is as discussed on Slack to enable PAF in TH we need to have it enable by default.
Is this wrong understanding?